woocommerce/woocommerce-ios
# Separate list of custom and predefined packages in view model

## Description


- Separated list of custom and predefined packages in view model.
- Added ability to remove packages from WooSavedPackagesSelectionViewModel (later on will be used to remove from backend also), so it can be used with "trash" button in swipe action

## Testing information


1. Install and set up the Woo Shipping extension on your store.
2. Build and run the app with the `revampedShippingLabelCreation` feature flag enabled.
3. Create an order with the processing status and at least one physical product.
4. In the order details, select "Create Shipping Label."
5. Tap "Select a Package" button
6. Switch to "Saved" tab
7. Swipe one of the items in the list and tap the trash icon. Item should be removed from the list
8. Tap/select one of the items in the list and "Add Package" button should be enabled

## Screenshots


https://github.com/user-attachments/assets/735e378c-d61d-4340-b6e5-8a743a14d71f

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
